### ISSUE TITLE
Define Terms in the CoC and FAQ

### ISSUE THREAD

**Issue by LordStreetguru** [ContributorCovenant/contributor_covenant#598]
In order for maintainers and communities to better understand this CoC you should be defining terms used within it. 

A short list.
Inclusive
Empathy
Trolling
"Other conduct which could reasonably be considered inappropriate in a professional setting"
"other behaviors that they deem inappropriate, threatening, offensive, or harmful."

In the FAQ
(with the obvious exception of hate groups.) which I assume is a reference to people who are not allowed to contribute.

Vague terms are not useful for a CoC that's nearing a full legal contract for contributing to or maintaining a project.

**Comment by bkstigler08** [comment]
Be on your Game or Be Gone!

**Comment by Atiwat519** [comment]
Good

**Comment by Chealer** [comment]
I have to agree about some of the list. I opened ticket #1457 specifically about "trolling". Professionalism is tracked in ticket #39.

Regarding the last case, the _Enforcement Responsabilities_ section contains:

> Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.

I would say that in volunteer contexts like most organizations using this CoC, it is simply unrealistic to expect whoever to take corrective action for any inappropriate or harmful action, so the issue is not just definition. That paragraph should be reviewed.

**Comment by LordStreetguru** [comment]
> I have to agree about some of the list. I opened ticket [#1457](https://github.com/EthicalSource/contributor_covenant/issues/1457) specifically about "trolling". Professionalism is tracked in ticket [#39](https://github.com/EthicalSource/contributor_covenant/issues/39).
> 
I did not expect this to ever come back, but we're in a new dawn with the hopeful death of DEI.

My position has become something like, if Terry A Davis couldn't have contributed to your codebase, you need to rethink your policies. A Terry clause if you will.
